From 7162088ec80add0e83d1073b67001546be3d0d8d Mon Sep 17 00:00:00 2001
From: Nikolai Kondrashov <Nikolai.Kondrashov@redhat.com>
Date: Thu, 23 Oct 2014 13:56:46 +0300
Subject: [PATCH 1/1] raddb: Update triggers in trigger.conf

Update trigger.conf's available triggers and comments to correspond to
actual code.
---
 raddb/trigger.conf | 23 ++++++++++++-----------
 1 file changed, 12 insertions(+), 11 deletions(-)

diff --git a/raddb/trigger.conf b/raddb/trigger.conf
index 44f9f36..413a182 100644
--- a/raddb/trigger.conf
+++ b/raddb/trigger.conf
@@ -222,11 +222,8 @@ trigger {
 			# A connection to the DB has been closed
 			close = "${snmptrap}::serverModuleConnectionDown ${args}"
 
-			# Failed to open a new connection to the DB
-			fail = "${snmptrap}::serverModuleConnectionFail ${args}"
-
-			# There are no DB handles available.
-			none = "${snmptrap}::serverModuleConnectionNone ${args}"
+			# The module has been HUP'd via radmin
+			hup = "${snmptrap}::serverModuleHup ${args}"
 		}
 
 		# The SQL module
@@ -243,12 +240,13 @@ trigger {
 			# Failed to open a new connection to the DB
 			fail = "${snmptrap}::serverModuleConnectionFail ${args}"
 
-			# There are no DB handles available.
-			none = "${snmptrap}::serverModuleConnectionNone ${args}"
+			# The module has been HUP'd via radmin
+			hup = "${snmptrap}::serverModuleHup ${args}"
 		}
 
-		# You can use the same opn / close / fail / none triggers for
-		# any module which uses the "pool" directive.
+		# You can also use connection pool's start/stop/open/close triggers
+		# for any module which uses the "pool" section, here and under
+		# pool.trigger in module configuration.
 	}
 }
 
@@ -267,10 +265,9 @@ trigger {
 # home_server_pool.fallback
 # home_server_pool.normal
 # modules.*.hup
-# modules.ldap.fail
+# modules.ldap.timeout
 # modules.sql.close
 # modules.sql.fail
-# modules.sql.none
 # modules.sql.open
 # server.client.add
 # server.max_requests
@@ -278,3 +275,7 @@ trigger {
 # server.signal.term
 # server.start
 # server.stop
+# server.thread.max_threads
+# server.thread.start
+# server.thread.stop
+# server.thread.unresponsive
-- 
2.1.1

